core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -8,6 +8,17 @@ import {
   SsrOptimizationOptions,
 } from './ssr-optimization-options';
 
+export type SsrCallbackFn = (
```

Comment:
Adding this comment to a random place - as agreed, we will re-add a log statement.